lib/auth/grpcserver.go

```diff
@@ -367,6 +367,62 @@ func (g *GRPCServer) GetUsers(req *proto.GetUsersRequest, stream proto.AuthServi
 	return nil
 }
 
+// CreateUser inserts a new user entry in a backend.
+func (g *GRPCServer) CreateUser(ctx context.Context, req *services.UserV2) (*empty.Empty, error) {
```

Comment:
looks like authz is missing in all of these methods.
RPCs should follow the general path: `GRPCServer -> ServerWithRoles -> Server`